src/Tes.RunnerCLI/Commands/CommandFactory.cs

```diff
@@ -76,53 +93,55 @@ internal static Command CreateCommand(string optName, string optDescription, par
         {
             var cmd = new Command(optName, optDescription);
 
-            var allOptions = CreateGlobalOptionList();
-
-            allOptions.AddRange(options);
-
-            foreach (var option in allOptions)
+            foreach (var option in options)
             {
                 cmd.AddOption(option);
             }
 
             return cmd;
         }
 
-        private static List<Option> CreateGlobalOptionList()
```

Comment:
I thinks method clearly states what are the global options in the CLI, trying to abstract that as collection upstream seems less readable to me.

Reply:
The collection is named `GlobalOptions`. Seems very readable to me.